bin/test-all.js

```diff
@@ -0,0 +1,21 @@
+const updateTest = require('./test-entry-update.js');
+const creationTest = require('./test-entry-creation.js');
+
+const cmaToken = process.env.CMA_TOKEN;
+const spaceId = process.env.SPACE_ID;
+const cdaToken = process.env.CDA_TOKEN;
+const existingEntryId = process.env.EXISTING_ENTRY_ID;
+
+require.main === module && run();
+
+async function run () {
+  await creationTest.run({ cmaToken, cdaToken, spaceId });
```

Comment:
I think we should not manually invoke every test. This means any test we add we need to import it and run it manually. 
I think we should run all the files either via an npm script or define it in directly the CI.

Reply:
had similar doubts writing it, undone